plugins/slack_app/README.md

```diff
@@ -0,0 +1,12 @@
+# Slack App Plugin
+
+This plugin adds a `/deploy` command to Slack which creates Samson deployments.
+To activate:
+
+1. Go over to Slack and create a [new app](https://api.slack.com/apps/new). Many of the fields are completely up to you, but these are important:
+  - Grab the Client ID and secret and verification token, and record them in your environment as `SLACK_CLIENT_ID`, `SLACK_CLIENT_SECRET`, and `SLACK_VERIFICATION_TOKEN`.
```

Comment:
That's odd. Mine looks like this:

![image](https://cloud.githubusercontent.com/assets/39902/17377176/92dc8e58-596d-11e6-84c3-d36fd4c0702c.png)

Maybe you have to enable interactive messages or slash-commands before it shows up?